File: 15. Animations/03. Zoom-Up Animation/TaskManager-Angular/src/app/sign-up-view-model.ts
export class SignUpViewModel
{
    personName: any;
    email: string | any;
    mobile: string | any;
    dateOfBirth: string | any;
    password: string | any;
    gender: string | any;
    countryID: number | any;
    receiveNewsLetters: boolean | any;
    skills: any;

    constructor(personName: any = null,
        email: string | any = null,
        mobile: string | any = null,
        dateOfBirth: string | any = null,
        password: string | any = null,
        gender: string | any = null,
        countryID: number | any = null,
        receiveNewsLetters: boolean = false,
        skills: any = null,)
    {
        this.personName = personName;
        this.email = email;
        this.mobile = mobile;
        this.dateOfBirth = dateOfBirth;
        this.password = password;
        this.gender = gender;
        this.countryID = countryID;
        this.receiveNewsLetters = receiveNewsLetters;
        this.skills = skills;
    }
}

File: Old/08. Component Communication/03. ViewChild/MvcTaskManager-AspNetCore2.1/MvcTaskManager/ViewModels/SignUpViewModel.cs
using MvcTaskManager.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MvcTaskManager.ViewModels
{
    public class SignUpViewModel
    {
        public PersonFullName PersonName { get; set; }
        public string Email { get; set; }
        public string Mobile { get; set; }
        public string DateOfBirth { get; set; }
        public string Password { get; set; }
        public string Gender { get; set; }
        public int CountryID { get; set; }
        public bool ReceiveNewsLetters { get; set; }
        public List<Skill> Skills { get; set; }
    }

    public class PersonFullName
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
    }
}


